code/edgeidentity/src/test/java/com/adobe/marketing/mobile/edge/identity/IdentityPropertiesTests.java

```diff
@@ -301,9 +467,7 @@ public void test_clearPrimaryECID_alsoClearsSecondaryECID() {
 		props.setECID(null);
 
 		// verify
-		assertEquals(0, flattenMap(props.toXDMData(false)).size());
-		assertNull(props.getECIDSecondary());
```

Comment:
I think we should still have these null asserts for this and all the other tests

Reply:
Thanks for pointing that out.  I add the assetNull back.